realm/realm-library/src/main/java/io/realm/RealmObject.java

```diff
@@ -326,4 +326,12 @@ void setTableVersion() {
             currentTableVersion = row.getTable().version();
         }
     }
+
+    @SuppressWarnings("UnusedParameters")
+    static Object get(RealmObject obj, String fieldName) {
+        return null;
```

Comment:
Perhaps add a comment saying this is just used to keep the annotation processor happy,